packages/bpk-component-mobile-scroll-container/src/BpkMobileScrollContainer.js

```diff
@@ -190,6 +197,12 @@ class BpkMobileScrollContainer extends Component<Props, State> {
       classNames.push(this.state.scrollIndicatorClassName);
     }
 
+    const scrollerClassNames = !showScrollbar
```

Comment:
Instead of doing the logic you can do the following for a cleaner implementation

```suggestion
    const scrollerClassNames = getClassName(
        'bpk-mobile-scroll-container__scroller',  
        showScrollbar && 'bpk-mobile-scroll-container__showScrollbar'
     );
```